ydb/core/protos/kqp.proto

```diff
@@ -111,17 +111,17 @@ message TQueryRequest {
 }
 
 message TKqpPathIdProto {
-    optional uint32 OwnerId = 1;
-    optional uint32 TableId = 2;
+    optional uint64 OwnerId = 1;
+    optional uint64 TableId = 2;
```

Comment:
The change from uint32 to uint64 produces binary compatible protobufs. See the docs of both [proto2](https://protobuf.dev/programming-guides/proto2/#updating) and [proto3](https://protobuf.dev/programming-guides/proto3/#updating), they have the same line on this topic:

```
int32, uint32, int64, uint64, and bool are all compatible – this means you can change a field from one of these types to another without breaking forwards- or backwards-compatibility
```